pudim.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Receitas da Hannah</title>
+        <link rel="stylesheet" href="pudim.css">
+    </head>
+    <body>
+        <h1>Receitas da Hannah</h1>
+        <h2>Join me on a flavor filled adventure!</h2>
+        <div class="boxes">
+            <a id="home" href="home.html">Home</a>
+            <a id="about" href="about.html">About</a>
+            <a id="brig" href="brigadeiro.html">Brigadeiro</a>
+            <a id="tapioca" href="tapioca.html">Tapioca</a>
+            <a id="pudim" href="a">Pudim</a>
+        </div>
+        <h2>Pudim</h2>
+        <img src="https://milestone-1.netlify.app/images/caprese.jpeg">
+        <p>This is a simpler version of what many people know as flan.</p>
+        <h4>Ingredients</h4>
+        <div>
+            <ul>
+                <li>sugar</li>
+                <li>hot water</li>
+                <li>condensed milk</li>
+                <li>milk</li>
+                <li>eggs</li>
+            </ul>
+
```

Comment:
it would be good to add an instructions h4 to visually separate your ingredients from your instructions